src/com/sun/ts/tests/signaturetest/signature-repository/jakarta.servlet.jsp.sig_3.0_se11

```diff
@@ -86,10 +86,10 @@ meth public abstract <%0 extends java.lang.Object> java.util.concurrent.Schedule
 meth public abstract java.util.concurrent.ScheduledFuture<?> schedule(java.lang.Runnable,jakarta.enterprise.concurrent.Trigger)
 
 CLSS public abstract interface jakarta.enterprise.concurrent.ManagedTask
-fld public final static java.lang.String IDENTITY_NAME = "javax.enterprise.concurrent.IDENTITY_NAME"
-fld public final static java.lang.String LONGRUNNING_HINT = "javax.enterprise.concurrent.LONGRUNNING_HINT"
+fld public final static java.lang.String IDENTITY_NAME = "jakarta.enterprise.concurrent.IDENTITY_NAME"
```

Comment:
I see this file needs to be regenerated, I will do that in a separate change. This was not picked since the jdk11 run has not started. Thanks for pointing this.